apis/runtime/v1alpha1/api.proto

```diff
@@ -0,0 +1,158 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// To regenerate api.pb.go run scripts/generate_runtime.sh
+syntax = "proto3";
```

Comment:
for the docker proxy, we need to transfer the http request to grpc request?  I think it's a little complicated.

Reply:
the request body has no difference between grpc and http when talking to RuntimeHookServer,  so  it should NOT be more complicated under docker scenario?

Reply:
> the request body has no difference between grpc and http when talking to RuntimeHookServer, so it should NOT be more complicated under docker scenario?

the request body has no difference between grpc and http when talking to RuntimeHookServer, so it should NOT be more complicated under docker scenario?

Reply:
ok